src/schema/comments.ts

```diff
@@ -383,6 +390,7 @@ export const resolvers: IResolvers<any, Context> = {
             postId,
             userId: ctx.userId,
             content,
+            content_html: markdown.render(content),
```

Comment:
I think we can use TypeORM listeners for better coverage:
https://typeorm.io/#/listeners-and-subscribers/beforeinsert
We can use before insert and before update

Reply:
Oh this is really cool! Will adjust to this indeed 👏